extra/js/ga.js

```diff
@@ -0,0 +1,4 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+gtag('config', 'UA-69270713-5');
```

Comment:
Was google analytics not included inside other static assets (e.g. `index.html`) previously?

Reply:
From `mkdocs-s3-publish-action` the sync step has been removed due to minio client, so we have the next step for that. It works better this way.

Reply:
GA added to all pages automatically

Reply:
Seems like a duplicate with https://github.com/Exabyte-io/documentation/pull/218/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R39 below - do we need to add the ID in both places??